src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -3396,4 +3399,110 @@ void subscribeExclusiveTopicAndPattern(VertxTestContext context) throws Throwabl
         context.completeNow();
         assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
     }
+
+    @Test
+    void consumerDeletedAfterInactivity(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String topic = "consumerDeletedAfterInactivity";
+        kafkaCluster.createTopic(topic, 1, 1);
```

Comment:
I don't think that we need all the complexity of creating a topic, subscribing, etc etc ...
We have just to create the consumer because the timeout checker will check anyway; we can just simplify the test code in this way.